helm-charts/karpenter-resources/values.yaml

```diff
@@ -5,9 +5,9 @@ clusterName: test-cluster
 # EC2NodeClass specific values
 ec2NodeClass:
   enabled: true
-  amiFamily: Bottlerocket
+  amiFamily: AL2
```

Comment:
we should not use AL2 as a default anymore - either Bottlerocket or AL2023

Reply:
If we are updating patterns, then we should update to either Bottlerocket or AL2023 - AL2 is reaching the end of support

Reply:
@bryantbiggs makes sense to me ... @vara-bonthu what do you think? Testing other blueprints with AL2023 or Bottlerocket will take time, and as we're using versioning, we won't break any existing DoEKS blueprint but we would need to let other owners know about it (I can help with that).

As soon as we have an agreement, I'll push a change to use Bottlerocket as default (or AL2023 if there's a preference on that).

Reply:
@vara-bonthu I've pushed a new commit to have Bottlerocket as a default amiSelectorTerms